platform/darwin/src/NSNumber+MGLStyleAttributeAdditions_Private.hpp

```diff
@@ -0,0 +1,15 @@
+#import <Foundation/Foundation.h>
+
+#import "MGLStyleAttributeValue.h"
+
+#include <mbgl/style/property_value.hpp>
+
+@interface NSNumber (MGLStyleAttributeAdditions_Private) <MGLStyleAttributeValue>
+
+- (mbgl::style::PropertyValue<bool>)mbgl_booleanPropertyValue;
+
++ (NSNumber *)mbgl_booleanWithPropertyValue:(mbgl::style::PropertyValue<bool>)propertyValue;
```

Comment:
This is fine, but if you ever find it tedious to support multiple `-mbgl_…withPropertyValue:` methods, it might be simpler to have a single `-mbgl_numberWithPropertyValue:` method that takes an `mbgl::style::PropertyValue<MGLNSNumberConvertible>`? `MGLNSNumberConvertible` would be typedef’d to `variant<bool, float, uint32_t, int32_t>`.